hangman/mix.exs

```diff
@@ -14,9 +14,8 @@ defmodule Hangman.Mixfile do
 
   def application do
     [
-      extra_applications: [
-        :logger
-      ]
+      extra_applications: [ :logger ],
+      mod: { Dictionary.Application, [] }
```

Comment:
# -5

This should be Hangman.Application: the Dictionary starts itself.
As a result of this, I get 

~~~
** (Mix) Could not start application hangman: Dictionary.Application.start(:normal, []) returned an error: already started: #PID<0.208.0>  
~~~

when I try to start Hangman—the code doesn't run.

I fixed this and continued to grade.